### DISCUSSION TITLE
floating windows resize/move at a different increment than tiled windows

### DISCUSSION THREAD

**Discussion by Merulox** [i3/i3#4974] [Q&A]
in my config, I have set the resize increment set to 5px, which works, but only for tiled windows-- floating windows move and resize at the increment of what seems to be 1px, regardless of what I put.
```
#resize 
bindsym $mod+shift+Right  resize shrink width 5 px or 5 ppt
bindsym $mod+shift+Up resize grow height 5 px or 5 ppt
bindsym $mod+shift+Down resize shrink height 5 px or 5 ppt
bindsym $mod+shift+Left  resize grow width 5 px or 5 ppt
```

**Comment by Airblader** [comment]
Are you resizing with the key binding, or with the mouse?

**Reply by Merulox** [reply]
with the key binding
